### DISCUSSION TITLE
VS Code Snippet Pack for lion

### DISCUSSION THREAD

**Discussion by bagherani** [ing-bank/lion#2074] [Open for contributions]
Hi dear maintainers.

I'm interested in contributing to the Lion UI Library by creating a VS Code snippet pack specifically designed for Lion UI components. I believe this would greatly enhance the development experience for users of the library.

I'd like to gather your suggestions and ideas on which parts of the Lion UI library should be covered in these snippets. As the repository owner and expert on the library, your input would be invaluable in creating a comprehensive and useful snippet pack.

![image](https://github.com/ing-bank/lion/assets/12948334/0f9dff73-138c-4286-ae7e-b7895c891c02)

Thank you for your time and consideration. I'm looking forward to your response.

**Comment by gerjanvangeest** [comment]
Hi @bagherani,

Lion is a white labeled base layer designed to be extended, so it should return something like:

```js
import { LionInput } from '@lion/ui/input.js';

export class MyInput extends LionInput {...}
```
